dev-client/src/screens/HomeScreen/components/TemporaryLocationCallout.tsx

```diff
@@ -85,8 +85,7 @@ export const TemporaryLocationCallout = ({
         <LatLngDetail isCurrentLocation={isCurrentLocation} coords={coords} />
       }
       buttons={<CloseButton onPress={closeCallout} />}
-      width="90%"
-      maxWidth="350px"
+      width="350px"
```

Comment:
I forget, is there a minimum screen width we support? Just wanna make sure this is smaller than that. (I feel like @CourtneyLee333 would know this)

Reply:
React Native requires iOS 13. The oldest device it supports is an iPhone SE (2015), which is 320px × 568px. So this would be too large for that phone (but no other supported iOS device).